testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/MultinodeHiveCaching.java

```diff
@@ -72,7 +72,7 @@ public void extendEnvironment(Environment.Builder builder)
     {
```

Comment:
Why `standardMultinode` is not used here?

Reply:
`standardMultinode` creates a single worker - while `MulitnodeHiveCaching` requires creating more than one workers

Reply:
Maybe we should make `standardMultinode` to be parametric. Anyway, out of the scope of this PR.